### DISCUSSION TITLE
Language code for Czech language

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#467] [Team Posts]
Dear all,

@DanD222 is making significant progress translating the app to Czech. Regretfully, I learned today that the Czech language code is not CZ (as I had guessed) but CS. Just so you know- I have changed all files accordingly.

@DanD222, please update and make sure you are working on the correct file (enroute_cs.ts). I apologize for the inconvenience!

Best,

Stefan.

**Comment by DanD222** [comment]
@kebekus No problem, will do; thanks for the heads up.
